### DISCUSSION TITLE
Is it compatible with Django Rest Framework?

### DISCUSSION THREAD

**Discussion by v3ss0n** [bhch/django-jsonform#88] [General]
Checking if possible to integrated the fields in DRF>

**Comment by bhch** [comment]
Did you try it? What issues did you find?

Theoretically, it should work with DRF since it's essentially just a `JSONField`.

**Comment by v3ss0n** [comment]
thanks i am going to give a try

**Reply by Mte90** [reply]
Did you try? My guess is that you pass the same values only in the body or with url parameters.
